luda-editor/luda-editor-client/src/app/sequence_list/list_item.rs

```diff
@@ -0,0 +1,84 @@
+use super::{
+    events::SequenceListEvent,
+    rounded_rectangle::RoundedRectangleColor,
+    types::{RenderingTreeRow, RenderingTreeRows, SequenceLoadStateDetail},
+    SequenceList, BUTTON_HEIGHT, MARGIN,
+};
+use crate::app::sequence_list::SPACING;
+use namui::{render, Wh};
+
+impl SequenceList {
+    pub fn render_list_item(&self, width: f32, title: &String) -> RenderingTreeRow {
+        let path = format!("sequence/{}", title);
+        let element_width = width - 2.0 * MARGIN;
+        let button_wh = Wh {
+            width: element_width,
+            height: BUTTON_HEIGHT,
+        };
+        let title_load_state = self.sequence_load_state_map.get(&path);
```

Comment:
I have no idea that it's the good idea to know whether the list is open or not using `sequence_load_state_map`.
`sequence_load_state_map` is the map to check about sequence loading state.
But you used it to determine whether the list is open or not in here.
One data for two purpose would make code complex.

I believe you at least hide this in function and manage that function. or just get information from parameter of `render_list_item`.

Do you have any idea to improve this?